irods/manager/data_object_manager.py

```diff
@@ -523,6 +530,8 @@ def open(
         allow_redirect=client_config.getter("data_objects", "allow_redirect"),
         **options
     ):
+        if buffering < 0:
```

Comment:
always true?   since set to -1 just above?

Reply:
The `buffering = -1` was in a parameter declaration so, it means a parameter named `buffering` with a default value of 1.  Admittedly an ambiguity from a human standpoint, and the reason many early languages like Pascal chose `:=` instead of `=` for their assignment operators.

Reply:
> ... so here it means a parameter named `buffering` with a default value of 1.

I don't understand. The default value is `-1`, based on the function signature. Am I misunderstanding something?

What values are considered valid for the `buffering` parameter?